protocols/ping/src/handler.rs

```diff
@@ -258,11 +258,11 @@ impl ConnectionHandler for Handler {
             match fut.poll_unpin(cx) {
                 Poll::Pending => {}
                 Poll::Ready(Err(e)) => {
-                    log::debug!("Inbound ping error: {:?}", e);
+                    tracing::debug!("Inbound ping error: {:?}", e);
                     self.inbound = None;
                 }
                 Poll::Ready(Ok(stream)) => {
-                    log::trace!("answered inbound ping from {}", self.peer);
+                    tracing::trace!("answered inbound ping from {}", self.peer);
```

Comment:
I wonder whether we should remove `self.peer` from log statements like this.

On the one hand, for users that use `tracing`, this will be redundant information. On the other hand, non-users of `tracing` will be missing this context.

I am tempted to take a more opinionated approach here and say that, if you want informative logs, you should use `tracing` and we make those logs less noisy by removing this redundant information.

Thoughts @mxinden?